src/lib/y2network/widgets/interfaces_table.rb

```diff
@@ -143,11 +142,15 @@ def create_description
         summary.new(value, config).text
       end
 
-      # Returns a friendly name for a given interface
+      # Returns the connection description if given or the interface friendly name if not
       #
       # @param interface [Interface] Network interface
-      # @return [String] Friendly name for the interface (description or name)
-      def friendly_name(interface)
+      # @param conn [ConnectionConfig::Base] Connection configuration
+      # @return [String] Connection description if given or the friendly name for the interface (
+      #   description or name) if not
+      def description_for(interface, conn)
```

Comment:
```interface``` is a bit unnecessary ... we should be able to get it from the connection config. Or vice versa. Positive of such approach would be avoiding theoretical mismatch between interface and connection config.

Reply:
Yeah, I already thought about using the connection interface but as the interface was there and in fact the connection was find based on the in the interface name just passed it by parameter. It is not a public API and preferred to be conservative in this case according to last issues :stuck_out_tongue_winking_eye: